src/GameSession.js

```diff
@@ -1,27 +1,62 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import Gameboard from './Gameboard.js';
 import randomGen from './funcLib/RandomGen.js';
 import wordProcessor from './funcLib/WordProcessor.js';
 import wordImporter from './funcLib/WordImporter.js';
 import PlayAgainButton from './PlayAgainButton.js';
 import { Col, Container, Row } from 'react-bootstrap';
-import { Link } from 'react-router-dom';
+import checkForBingo from './funcLib/CheckForBingo';
 
 export default function GameSession() {
+  const [moves, setMoves] = useState(0);
+  const [isBingoed, setBingoed] = useState(false);
+  const [dauberedTiles, setDauberedTiles] = useState([]);
+  const [gamesStarted, setGamesStarted] = useState([1]);
   const randInts = randomGen(24);
   let words = wordImporter();
-  const randWords = wordProcessor(words, randInts);
+  const [randWords, setRandWords] = useState(wordProcessor(words, randInts));
+
+  function handleTileClick(e) {
+    let id = e.currentTarget.id;
+    if(id !== null) {
+      setDauberedTiles((prev) =>{
+        let modDauberedTiles = prev;
+        modDauberedTiles[id] = true;
+        return modDauberedTiles;});
+      setMoves(moves + 1);
+    }
+  }
+
+  function restartGame() {
+    setGamesStarted(gamesStarted + 1);
+    setMoves(0);
+    setDauberedTiles([]);
```

Comment:
I think this is causing the FREE space to not be daubered. I'm fine with that and two things will need to happen 1) I'll need to verify the Bingo calculators and maybe update the test data, and 2) We will need to update the Bingo instructions to indicate it is up to the player to click FREE in order to count it.

Reply:
Whoops missed this but it should be relatively easy to fix. I think I'd prefer to fix it so that the FREE space is daubered at start as originally intended, if that is okay!

Reply:
Yes!

Reply:
Done!